app/src/main/java/org/groundplatform/android/ui/home/mapcontainer/jobs/LoiJobSheet.kt

```diff
@@ -235,6 +235,45 @@ fun PreviewModalContentsWhenJobHasTasks() {
       created = auditInfo,
       lastModified = auditInfo,
       geometry = Point(coordinates = Coordinates(lat = 20.0, lng = 20.0)),
+      isPredefined = false,
+    )
+  AppTheme {
+    ModalContents(loi = loi, canUserSubmitData = true, submissionCount = 20, onCollectClicked = {})
+  }
+}
+
+@SuppressLint("UnrememberedMutableState")
+@Composable
+@Preview(showBackground = true)
+@ExcludeFromJacocoGeneratedReport
+fun PreviewModalContentsWhenJobHasTasksAndIsPredefined() {
+  val loi =
+    LocationOfInterest(
+      id = "1",
+      surveyId = SURVEY_ID,
+      job =
+        Job(
+          id = "job",
+          style = Style(color = "#4169E1"),
```

Comment:
Can we use theme color as well

Reply:
This is only used for previewing purposes.